### PR TITLE
Add validation to preboot configuration

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

**Comment by szok** [comment]
:ok:

File: src/ralph/deployment/tests/test_forms.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import unittest
+from django.test import TestCase
+
+from ralph.deployment.forms import PrebootConfigurationForm
+
+
+class TestForm(TestCase):
+    def test_invalid_configuration(self):
+        form_data = {
+            'configuration': '{% foo bar %} wrong template tag',
```
Comment (ghost) [inline src/ralph/deployment/tests/test_forms.py:10]: you mean non-existant?

Comment (ar4s) [reply src/ralph/deployment/tests/test_forms.py:10]: yes

**Comment by ghost** [comment]
:ok:

**Pull request merged**